endpoints/profiles.md

```diff
@@ -7,31 +7,173 @@ The profile contains the contact information as well as the branding information
 ## Supported Actions
 
 * [Get Profiles](#get-profiles)
+* [Get Profile](#get-profile)
+* [Create Profile](#create-profile)
+* [Update Profile](#update-profile)
+* [Delete Profile](#delete-profile)
 
 ## Profile
 
 ```json
 {
-  "id": 8,
+  "id": 1,
   "profile_name": "Default Profile",
-  "full_name": "John Smith",
+  "photo": "https://...",
   "email": "john@example.com",
+  "first_name": "John",
+  "last_name": "Smith",
+  "middle_initial": "J",
+  "full_name": "John J Smith",
   "title": "Regional Manager",
-  "photo": "https://...",
-  "logo": "https://...",
-  "color": "#a20a0a",
-  "full_address": "6475 E Johns Xing, Johns Creek, GA, 30097, United States",
-  "address_line_1": "6475 E Johns Xing",
-  "address_line_2": "Suite B",
+  "company_name": "ABC Supply",
+  "address_line_1": "6475 East Johns Xing",
+  "address_line_2": "Apt. 202",
   "city": "Johns Creek",
   "state_region_province": "GA",
   "postal_code": "30097",
-  "country":"United States",
+  "country": "United States",
+  "country_code": "US",
+  "phone_1": "123-123-1234",
+  "phone_1_location": "Mobile",
+  "phone_2": "223-223-2234",
+  "phone_2_location": "Work",
+  "phone_3": "323-323-3234",
+  "phone_3_location": "Other",
+  "website": "https://john-smith.com",
+  "twitter": "https://twitter.com/example_user",
+  "linkedin": "https://linkedin.com/example_user",
+  "facebook": "https://facebook.com/example_user",
+  "blog": "https://blog.example.com",
+  "video_channel": "https://youtube.com/example_user",
+  "salutation": "Dear,",
+  "valediction": "Best wishes,",
+  "legal_footer": "Legal footer text",
+  "logo": "https://...",
+  "color": "#4B7B47",
+  "default": true
 }
 ```
+__*Note:*__ ```first_name``` , ```last_name```, and ```email``` are required.
+Any profile __created__ or __updated__ with ```email``` supplied will automatically receive a verification email.
+Accepted ```photo``` and ```logo``` formats __bmp__, __gif__, __jpg__, and __png__.
```

Comment:
formats are

Reply:
We need to specify how these files are actually serialized. I don't think we're sending binary data over the wire in this field.